### ISSUE TITLE
Premature Jellyfin library refresh

### ISSUE THREAD

**Issue by lazarevski** [itsToggle/plex_debrid#538]
I've managed to get the script working - I watchlist a movie/show on Trakt, the script picks it up, adds it on RD, rclone_rd syncs it to a local folder, Jellyfin start a full library refresh. The issue is that the library refresh starts before rclone_rd had a chance to sync to the local folder. Any way to delay the library refresh for Jellyfin?

**Comment by GrabbenD** [comment]
Any updates?
Got similar issues :/

**Comment by lazarevski** [comment]
Nope. I abandoned the idea of using Jellyfin with RD. Stremio is nice. Too bad i cant have one player (Jellyfin) to watch both locally stored and RD content.
